Dev-Task - Validate incorrect value reference in playbook tasks in SDK (#2289)
demisto/demisto-sdk: CHANGELOG.md

* Added support for Indicator Types/Reputations in the **upload** command.
* Fixed an issue when running from a subdirectory of a content repo failed.
* Changing the way we are using XSIAM servers api-keys in ***test-content*** command.

> * Added a new validation to the **validate** command to check the playbook task's value references.

Reviewer: ```suggestion
* Added a new check to **validate**, making sure playbook task values are passed as references.
```
